File: _history/20260206/test/client_code/tests/user/login_with_form.py
"""
user/login_with_form.py
"""


def main(use):

    login_with_form = use('@@/user:login_with_form')

    result = login_with_form(dict(allow_cancel=True, allow_remembered=False, show_signup_option=False))

    

    print("result:", result)

File: _history/20260206/test/client_code/tests/client/user/login_with_form.js
/*
client/user/login_with_form.js
*/

export default async () => {

  const logout = await use("@@/user:logout");
  await logout()
  
  const login_with_form = await use("@@/user:login_with_form");

  const result = await login_with_form({allow_cancel: true})
  console.log('result:', result)

  
};
